src/item_factory.cpp

```diff
@@ -414,6 +414,18 @@ void Item_factory::finalize_post( itype &obj )
             }
         }
     }
+
+    if( !obj.ascii_picture.empty() ) {
+        std::vector<std::string> tmp_ascii_pic;
+        for( std::string line : obj.ascii_picture ) {
```

Comment:
You can do this trimming in place, without a new vector:
```suggestion
        for( std::string &line : obj.ascii_picture ) {
```
Now your code edits the line directly.